src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11085,7 +11085,9 @@ private ExpressionSyntax ParseExpressionContinued(ExpressionSyntax unaryOrPrimar
                 }
 
                 if (IsExpectedAssignmentOperator(operatorToken.Kind))
-                {
+                    return ParseAssignmentOperator(operatorExpressionKind, leftOperand, operatorToken);
+
+#if false // extracted to 'ParseAssignmentOperator'
```

Comment:
I was getting horrific diffs when deleting the original local functions so trying to wait until later to actually do it.